src/Orleans/Core/GrainFactory.cs

```diff
@@ -290,6 +405,35 @@ internal static TGrainInterface GetSystemTarget<TGrainInterface>(GrainId grainId
         #endregion
 
         #region Utility functions
+
+        /// <summary>
+        /// Ensures that generated code reflects the current runtime environment.
+        /// </summary>
+        private static void EnsureCodeGenerationIsCurrent()
+        {
+            // Get the code generator if it is available.
+            if (codeGenerator == null)
```

Comment:
You're right, I was lazy to not use it :stuck_out_tongue: I'll fix this.
Here's my current draft replacement for this method:
```c#
/// <summary>
/// Ensures code the the <paramref name="input"/> assembly has been generated and loaded.
/// </summary>
private static void GenerateAndCacheForAssembly(Assembly input)
{
    var codeGen = CodeGenerator.Value;
    if (codeGen != null)
    {
        codeGen.GenerateAndCacheForAssembly(input);
    }
}
```

What do you think?